### PR TITLE
Issue #15449: partially excluded rule712paragraphs from google-java-format.sh

### PR DESCRIPTION
**Pull request by Zopsss**
#15449

Test file: https://github.com/checkstyle/checkstyle/blob/master/src/it/java/com/google/checkstyle/test/chapter7javadoc/rule712paragraphs/ParagraphsTest.java

### CODE REVIEW

File: .ci/google-java-format.sh [hunk 1]
```diff
@@ -6,7 +6,10 @@ JAR_PATH="$1"
 
 INPUT_PATHS=($(find src/it/resources/com/google/checkstyle/test/ -name "Input*.java" \
     | sed "s|src/it/resources/com/google/checkstyle/test/||" \
-    | grep -v "rule711generalform" | grep -v "rule712paragraphs" | grep -v "rule713atclauses" \
+    | grep -v "rule711generalform" \
+    | grep -v "rule712paragraphs/InputIncorrectRequireEmptyLineBeforeBlockTagGroup.java" \
+    | grep -v "rule712paragraphs/InputIncorrectJavadocParagraph.java" \
```
Comment (Zopsss) [inline .ci/google-java-format.sh:11]: after formatting tests of these two files were failing, so I had to grep them.

**Comment by romani** [comment]
yes, formatter do extra empty lines, or make it as single line:
```
✔ ~/java/github/romani/checkstyle [Zopsss/15549-rule712paragraphs L|✚ 2⚑ 2] 
06:37 $ git diff
diff --git a/.ci/google-java-format.sh b/.ci/google-java-format.sh
index 6f23aac..d0b7acb 100755
--- a/.ci/google-java-format.sh
+++ b/.ci/google-java-format.sh
@@ -8,7 +8,6 @@ INPUT_PATHS=($(find src/it/resources/com/google/checkstyle/test/ -name "Input*.j
     | sed "s|src/it/resources/com/google/checkstyle/test/||" \
     | grep -v "rule711generalform" \
     | grep -v "rule712paragraphs/InputIncorrectRequireEmptyLineBeforeBlockTagGroup.java" \
-    | grep -v "rule712paragraphs/InputIncorrectJavadocParagraph.java" \
     | grep -v "rule713atclauses" \
     | grep -v "rule73" \
     | grep -v "rule64" \
diff --git a/src/it/resources/com/google/checkstyle/test/chapter7javadoc/rule712paragraphs/InputIncorrectJavadocParagraph.java b/src/it/resources/com/google/checkstyle/test/chapter7javadoc/rule712paragraphs/InputIncorrectJavadocParagraph.java
index b3ca6ce..e35fd8b 100644
--- a/src/it/resources/com/google/checkstyle/test/chapter7javadoc/rule712paragraphs/InputIncorrectJavadocParagraph.java
+++ b/src/it/resources/com/google/checkstyle/test/chapter7javadoc/rule712paragraphs/InputIncorrectJavadocParagraph.java
@@ -5,8 +5,10 @@ package com.google.checkstyle.test.chapter7javadoc.rule712paragraphs;
 //  '\<p\> tag should be preceded with an empty line.'
 /**
  * Some Javadoc.
+ *
+ * <p>/^ WARN/ Some Javadoc.
+ *
  * <p>
- * /^ WARN/ Some Javadoc.<p>
  */
 // 2 violations 2 lines above:
 //  '\<p\> tag should be placed immediately before the first word'
@@ -18,9 +20,11 @@ class InputIncorrectJavadocParagraph {
   //  '\<p\> tag should be placed immediately before the first word'
   //  '\<p\> tag should be preceded with an empty line.'
   /**
-   * Some Javadoc.<p>
+   * Some Javadoc.
    *
-   * <p>  Some Javadoc.
+   * <p>
+   *
+   * <p>Some Javadoc.
    *
    * @since 8.0
    */
```

**Review by romani — APPROVED** [review]
ok to merge.

**Review by rdiachenko — APPROVED** [review]
lgtm

**Pull request merged**